index.js

```diff
@@ -188,6 +206,7 @@ const image = {
 
     // We support the file type
     token.consume(sentToken, matches[1], function (err) {
+        // OK #noKidding: Node-sqlite3 bind the `this`, so ES6 syntax is not possible here
         if (err || this.changes !== 1) {
```

Comment:
Keep in mind though that with the suggestion of @joostverdoorn `this` will no longer be available for non-lambda callbacks as well. You could support this by using `callback.call` or `callback.apply` if you like to.